### ISSUE TITLE
Bug: pyassimp.load() only works as context-manager

### ISSUE THREAD

**Issue by umlaeute** [assimp/assimp#5671]
according to the sample-scripts, it is possible to load a scene with `scene = pyassimp.load(...)`: https://github.com/assimp/assimp/blob/35e4f1bf6492e11485336c1518485413c599e00e/port/PyAssimp/scripts/sample.py#L23-L31

unfortunately, this is not true, as `pyassimp.load()` unconditionally returns a context-manager.

e.g. running this script:
```python
import pyassimp
import sys
scene = pyassimp.load(sys.argv[1])
print(f"meshes={scene.meshes}")
```

gives the following error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: '_GeneratorContextManager' object has no attribute 'meshes'
```

otoh, using `pyassimp.load()` as a context manager works nicely:
```python
import pyassimp
import sys
with pyassimp.load(sys.argv[1]) as scene:
   print(f"meshes={scene.meshes}")
```

using context-managers is a great thing in scripts, but when working interactively, it's sometimes preferably to be able to get an object (for interactive inspection).

it would be great if `pyassimp.load()` could work the same as e.g. the standard `open()`: usable both as a context-manager or as an object-returning function.

**Comment by umlaeute** [comment]
obviously, this has been in PyAssimp since efbabf3b0daccb976639a3734fa9018c4b3e13fa

**Comment by tellypresence** [comment]
@umlaeute Is this still a problem?

**Comment by umlaeute** [comment]
yes

**Comment by tellypresence** [comment]
pyassimp issues tracked via epic #6177
